File: 0701_0800/0760/0760.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    n, v, l = (int(x) for x in f.readline().split())
    q.write("{:.2f}".format(l / v * 60 + sum(int(x) for x in f.readline().split()[1::2])))

File: 0701_0800/0760/0760.c
#include <stdio.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int n, v, l;
	double t;

	fscanf(f, "%d%d%d", &n, &v, &l);
	t = (double)l / v * 60;
	while (n--)
		fscanf(f, "%d%d", &v, &v), t += v;
	fprintf(q, "%.2f", t);
	return 0;
}
